runhouse/main.py

```diff
@@ -578,6 +594,93 @@ def status(
     _print_status(cluster_status, current_cluster)
 
 
+async def aget_clusters_from_den():
```

Comment:
why do we have to do this async?

Reply:
because we use `httpx_client = httpx.AsyncClient()`. We it use to be consistent with other calls we make to den from OSS. 

There is a a sync wrapper for `aget_clusters_from_den`.

Reply:
i think would be cleaner to just call `rns_client.session.get`, instead of setting up separate helpers and http client to manage this